libImaging/Resample.c

```diff
@@ -99,14 +99,16 @@ ImagingPrecompute(int inSize, int outSize, struct filter *filterp,
         return 0;
 
     // sizeof(double) should be greater than 0 as well
-    if (outSize > INT_MAX / (2 * sizeof(double)))
+    if (outSize > INT_MAX / (2 * sizeof(int)))
```

Comment:
Probably we don't need this check at all, because `sizeof(double)` > `sizeof(int)` and kmax can't be less than 3 (filterscale >= 1, kmax = filterp->support * filterscale * 2 + 1;